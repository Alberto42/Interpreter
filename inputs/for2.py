x = 0
i = 42
for i in range(1,10) {
    x = x + i
}