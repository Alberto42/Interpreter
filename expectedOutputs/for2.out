Ident "x" = VInt 55
Ident "i" = VInt 42


